src/track/playcounter.cpp

```diff
@@ -0,0 +1,22 @@
+#include "track/playcounter.h"
+
+
+void PlayCounter::updatePlayed(bool bPlayed) {
+    if (bPlayed) {
+        // increment the play count when play during the current session
```

Comment:
"when" is confusing here, because the code is not conditional. 
Does the comment add any value? 
We may consider to rename m_iTimesPlayed instead. 
How about "m_timesPlayedCurrentSession"